src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx

```diff
@@ -69,7 +69,7 @@ const WorkPackageEditDetails: React.FC<Props> = ({ users, control, errors }) =>
           />
         </Grid>
         <Grid item xs={12} md={4} sx={{ mt: 2, mb: 1 }}>
-          {statusSelect}
+          <Typography>{statusSelect}</Typography>
```

Comment:
the statusSelect is also a react component, so it doesn't need a Typography around it